src/components/TextField/TextField.tsx

```diff
@@ -132,21 +132,29 @@ class TextField extends React.PureComponent<CombinedProps, State> {
 
     this.state = {
       height: null,
-      focus: false,
+      focus: props.focused || false,
```

Comment:
We can probably leave this as
```suggestion
      focus: props.focused,
```

Reply:
True! I just did this for type "sanity" but I'll change it back.

Reply:
Oh, nevermind. Since `focused` is possibly `undefined`, we need to set a `boolean` default value.

Reply:
Sorry I thought it optional 😅